platform/linux-generic/odp_schedule_scalable_ordered.c

```diff
@@ -71,6 +71,7 @@ bool rwin_reserve(reorder_window_t *rwin, uint32_t *sn)
 	return true;
 }
 
+static
 void rwin_insert(reorder_window_t *rwin,
```

Comment:
The coding style of this file (and majority of code base) is to place "static" on the same line as rest of function definition. Please fix this up so that it remains consistent